engines/e_devcrypto.c

```diff
@@ -1234,3 +1263,22 @@ void engine_load_devcrypto_int()
     ENGINE_free(e);          /* Loose our local reference */
     ERR_clear_error();
 }
+
+#else
+
+static int bind_helper(ENGINE *e, const char *id)
+{
+    if ((id && (strcmp(id, engine_devcrypto_id) != 0))
+        || !open_devcrypto())
+        return 0;
+    if (!bind_devcrypto(e)) {
+        close_devcrypto();
+        return 0;
+    }
+    return 1;
+}
+
+IMPLEMENT_DYNAMIC_CHECK_FN()
+    IMPLEMENT_DYNAMIC_BIND_FN(bind_helper)
```

Comment:
Why the extra indentation here?

Reply:
I'm guilty of _copy & paste_ here, as I don't know the reason to do it. I've followed the majority in `engines`: `e_afalg.c`, `e_dasync.c`, and `e_ossltest.c` have the indentation, while `e_capi.c` and `e_padlock.c` do not.  I just now saw that 32c6985349ba134761f75a3f61814234d096a1df changed it in `e_capi.c`, so I'll remove it here as well.

Reply:
That indentation has happened because the line before doesn't have an ending semicolon, which throws off automatic source formatters.